src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star_desactived.png);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
Consider adding a fallback font-family for your icons in case the image fails to load, ensuring that some form of icon is still displayed.